Formula/packetq.rb

```diff
@@ -0,0 +1,26 @@
+class Packetq < Formula
+  desc "SQL-frontend to PCAP-files"
+  homepage "https://www.dns-oarc.net/tools/packetq"
+  url "https://github.com/DNS-OARC/PacketQ/archive/v1.3.1.tar.gz"
```

Comment:
@JCount on the packetq release page (https://github.com/DNS-OARC/PacketQ/releases), under Release 1.3.1 > Downloads, this is the exact link the developer provided.

looks to me like a lot of brew formulas are linking to github archive:
$ grep archive/ Formula/*